web/src/routes/MyPages/ProfilePage.tsx

```diff
@@ -128,23 +135,9 @@ const PurchaseButton = styled.div`
   }
 `;
 
-const initalState = {
-  point: '0',
-  email: 'test@gmail.com',
-  password: 'asdf1234!',
-  name: 'jsontest',
-  phoneNumber: '010-1234-5678',
-  address: '서울특별시 성북구 정릉동 국민대학교 기숙사',
-  emailDisabled: false,
-  passwordDisabled: false,
-  nameDisabled: false,
-  phoneNumberDisabled: false,
-  addressDisabled: false,
-};
-
-const formatPhoneNumber = (value: string): string => {
+const changePhoneNumber = (value: string): string => {
   const phone = value.replace(/[^0-9]/g, '');
```

Comment:
Okay `phone`. What is `phone`? This should be `phoneNumber`. The name `phone`, is more likely to some kind of abstract class.